client/me/security-2fa-disable/index.jsx

```diff
@@ -132,11 +132,11 @@ class Security2faDisable extends Component {
 			return (
 				<div className="security-2fa-disable__prompt">
 					<FormSectionHeading>
-						{ translate( 'Disable Two-Step Authentication' ) }
+						{ translate( 'Disable two-step authentication' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Disable Two Step Authentication' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)